Error building codimd server with longtext

Hello,

Trying to build current master, following the guide (), I've got this error:
  ERROR: Migration 20171009121200-longtext-for-mysql.js (or wrapper) didn't return a promise﻿
  [...] npm ERR! CodiMD@1.4.0 start: `sequelize db:migrate && node app.js` 

Steps followed :
* debian 9.9 up-to-date as of this evening
* install nodejs 10.0 LTS using official binary distribution
* install yarn
* install debian stable-packaged postgresql
* git clone https://github.com/codimd/server (2019-06-28, about 18:00 UTC)
* configure database settings properly
* npm start --production

Thank you!

--- a/lib/migrations/20171009121200-longtext-for-mysql.js
+++ b/lib/migrations/20171009121200-longtext-for-mysql.js
@@ -1,16 +1,16 @@
 'use strict'
 module.exports = {
-  up: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'content', { type: Sequelize.TEXT('long') })
-    queryInterface.changeColumn('Revisions', 'patch', { type: Sequelize.TEXT('long') })
-    queryInterface.changeColumn('Revisions', 'content', { type: Sequelize.TEXT('long') })
-    queryInterface.changeColumn('Revisions', 'lastContent', { type: Sequelize.TEXT('long') })
+  up: async function (queryInterface, Sequelize) {
+    await queryInterface.changeColumn('Notes', 'content', { type: Sequelize.TEXT('long') })
+    await queryInterface.changeColumn('Revisions', 'patch', { type: Sequelize.TEXT('long') })
+    await queryInterface.changeColumn('Revisions', 'content', { type: Sequelize.TEXT('long') })
+    await queryInterface.changeColumn('Revisions', 'lastContent', { type: Sequelize.TEXT('long') })
   },
 
-  down: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'content', { type: Sequelize.TEXT })
-    queryInterface.changeColumn('Revisions', 'patch', { type: Sequelize.TEXT })
-    queryInterface.changeColumn('Revisions', 'content', { type: Sequelize.TEXT })
-    queryInterface.changeColumn('Revisions', 'lastContent', { type: Sequelize.TEXT })
+  down: async function (queryInterface, Sequelize) {
+    await queryInterface.changeColumn('Notes', 'content', { type: Sequelize.TEXT })
+    await queryInterface.changeColumn('Revisions', 'patch', { type: Sequelize.TEXT })
+    await queryInterface.changeColumn('Revisions', 'content', { type: Sequelize.TEXT })
+    await queryInterface.changeColumn('Revisions', 'lastContent', { type: Sequelize.TEXT })
   }
 }

--- a/lib/migrations/20180209120907-longtext-of-authorship.js
+++ b/lib/migrations/20180209120907-longtext-of-authorship.js
@@ -1,13 +1,13 @@
 'use strict'
 
 module.exports = {
-  up: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'authorship', { type: Sequelize.TEXT('long') })
-    queryInterface.changeColumn('Revisions', 'authorship', { type: Sequelize.TEXT('long') })
+  up: async function (queryInterface, Sequelize) {
+    await queryInterface.changeColumn('Notes', 'authorship', { type: Sequelize.TEXT('long') })
+    await queryInterface.changeColumn('Revisions', 'authorship', { type: Sequelize.TEXT('long') })
   },
 
-  down: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'authorship', { type: Sequelize.TEXT })
-    queryInterface.changeColumn('Revisions', 'authorship', { type: Sequelize.TEXT })
+  down: async function (queryInterface, Sequelize) {
+    await queryInterface.changeColumn('Notes', 'authorship', { type: Sequelize.TEXT })
+    await queryInterface.changeColumn('Revisions', 'authorship', { type: Sequelize.TEXT })
   }
 }

--- a/lib/migrations/20180306150303-fix-enum.js
+++ b/lib/migrations/20180306150303-fix-enum.js
@@ -2,10 +2,10 @@
 
 module.exports = {
   up: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'permission', { type: Sequelize.ENUM('freely', 'editable', 'limited', 'locked', 'protected', 'private') })
+    return queryInterface.changeColumn('Notes', 'permission', { type: Sequelize.ENUM('freely', 'editable', 'limited', 'locked', 'protected', 'private') })
   },
 
   down: function (queryInterface, Sequelize) {
-    queryInterface.changeColumn('Notes', 'permission', { type: Sequelize.ENUM('freely', 'editable', 'locked', 'private') })
+    return queryInterface.changeColumn('Notes', 'permission', { type: Sequelize.ENUM('freely', 'editable', 'locked', 'private') })
   }
 }

